docs/journalist.rst

```diff
@@ -1,289 +1,295 @@
+key has two parts: a *public key* and a *private key*. The private
+key, used for decryption, stays on the *Journalist Workstation*. The
+public key, used for encryption, is copied to the *Secure Viewing
+Station*.
 
-Create GPG key for the journalist
----------------------------------
+.. todo:: This document recommends transferring documents from the
+          *SVS* to the *Journalist Workstation*, without any
+          discussion of the potential risks or mitigations that should
+          be taken when doing so. A section needs to be added on why
+          doing this could be risky, and what can be done to make this
+          situation better.
 
-Each journalist must have a personal GPG key that they use for
-encrypting files transferred from the *Secure Viewing Station* to their
-Journalist Workstation. The private key, used for decryption, stays on
-their Journalist Workstation. The public key, used for encryption, gets
-copied to the *Secure Viewing Station*.
+If you do not yet have a GPG key, follow the instructions for your
+operating system to set one up with GnuPG:
 
-.. todo:: This document recommends transferring documents from the *SVS*
-          to the Journalist Workstation, without any discussion of the
-          potential risks or mitigations that should be taken when
-          doing so. A section needs to be added on why doing this
-          could be risky, and what can be done to make this situation
-          better.
+- `GNU/Linux <https://www.gnupg.org/gph/en/manual.html#AEN26>`__
+- `Windows <http://gpg4win.org/>`__
+- `Mac OS X <https://support.gpgtools.org/kb/how-to/first-steps-where-do-i-start-where-do-i-begin>`__
 
-If a journalist does not yet have a GPG key, they can follow these
-instructions to set one up with GnuPG (GPG).
+Connecting to the *Journalist Interface*
+----------------------------------------
 
--  `GNU/Linux <https://www.gnupg.org/gph/en/manual.html#AEN26>`__
--  `Windows <http://gpg4win.org/>`__
--  `Mac OS
-   X <https://support.gpgtools.org/kb/how-to/first-steps-where-do-i-start-where-do-i-begin>`__
+Journalists viewing documents on SecureDrop must connect to the
+*Journalist Interface* using the `Tails operating system
+<https://tails.boum.org/>`__. Your administrator can help provide you
+with a Tails drive.
+
+.. important:: See our guide on setting up :doc:`Tails for the Admin
+          and Journalist Workstation <tails_guide>` before continuing.
+
+.. note:: The Tails OS makes using SecureDrop very different from
+          other computing experiences. The added layers of security
+          mean extra steps each time you want to login. With practice,
+          you will become increasingly comfortable with the process.
+	  
+Each journalist has an authenticated Tor hidden service URL for
+logging in to the *Journalist Interface*. This must be done using the
+Tails operating system. After visiting the ".onion" URL and clicking
```

Comment:
`After visiting the ".onion" URL and clicking` This order seems wrong I think. The Journalist should click the Journalist Interface icon on the desktop which will open Tor Browser to the correct .onion page.

Reply:
This whole section on flagging for reply seems overly-technical to me. Maybe let's just say that there could be several reasons to flag, e.g. a source looks promising?